irr/include/IAnimatedMesh.h

```diff
@@ -39,19 +36,9 @@ class IAnimatedMesh : public IMesh
 	virtual void setAnimationSpeed(f32 fps) = 0;
 
 	//! Returns the IMesh interface for a frame.
-	/** \param frame: Frame number as zero based index. The maximum
-	frame number is getFrameCount() - 1;
-	\param detailLevel: Level of detail. 0 is the lowest, 255 the
-	highest level of detail. Most meshes will ignore the detail level.
-	\param startFrameLoop: Because some animated meshes (.MD2) are
-	blended between 2 static frames, and maybe animated in a loop,
-	the startFrameLoop and the endFrameLoop have to be defined, to
-	prevent the animation to be blended between frames which are
-	outside of this loop.
-	If startFrameLoop and endFrameLoop are both -1, they are ignored.
-	\param endFrameLoop: see startFrameLoop.
-	\return Returns the animated mesh based on a detail level. */
-	virtual IMesh *getMesh(s32 frame, s32 detailLevel = 255, s32 startFrameLoop = -1, s32 endFrameLoop = -1) = 0;
+	/** \param frame: Frame number, >= 0, <= getMaxFrameNumber()
+	\return Returns the animated mesh for the given frame */
+	virtual IMesh *getMesh(f32 frame) = 0;
```

Comment:
I would prefer a short comment saying what happens when I request frame 1.5 and only 1 and 2 exist.
or are float frames even allowed?

Reply:
It linearly interpolates. Added a comment.